diff --git a/node_modules/react-native-os/LICENSE b/node_modules/react-native-os/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/README.md b/node_modules/react-native-os/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/RNOS.podspec b/node_modules/react-native-os/RNOS.podspec
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/android/build.gradle b/node_modules/react-native-os/android/build.gradle
old mode 100644
new mode 100755
index 7055b60..784681d
--- a/node_modules/react-native-os/android/build.gradle
+++ b/node_modules/react-native-os/android/build.gradle
@@ -44,5 +44,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
diff --git a/node_modules/react-native-os/android/gradle/wrapper/gradle-wrapper.jar b/node_modules/react-native-os/android/gradle/wrapper/gradle-wrapper.jar
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/android/gradle/wrapper/gradle-wrapper.properties b/node_modules/react-native-os/android/gradle/wrapper/gradle-wrapper.properties
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/android/gradlew b/node_modules/react-native-os/android/gradlew
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/android/gradlew.bat b/node_modules/react-native-os/android/gradlew.bat
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/android/src/main/AndroidManifest.xml b/node_modules/react-native-os/android/src/main/AndroidManifest.xml
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/android/src/main/java/com/peel/react/RNOS.java b/node_modules/react-native-os/android/src/main/java/com/peel/react/RNOS.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/android/src/main/java/com/peel/react/RNOSModule.java b/node_modules/react-native-os/android/src/main/java/com/peel/react/RNOSModule.java
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/index.js b/node_modules/react-native-os/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/ios/RNOS.h b/node_modules/react-native-os/ios/RNOS.h
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/ios/RNOS.m b/node_modules/react-native-os/ios/RNOS.m
old mode 100644
new mode 100755
diff --git a/node_modules/react-native-os/ios/RNOS.xcodeproj/project.pbxproj b/node_modules/react-native-os/ios/RNOS.xcodeproj/project.pbxproj
old mode 100644
new mode 100755
